<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeekReportMainFrame</class>
 <widget class="QWidget" name="WeekReportMainFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_min">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>0</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>16</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-image: url(:/images/min_first.png);
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{
background-image: url(:/images/min_after.png);
background-repeat:no-repeat;
background-position:center center;


}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>0</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>16</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-image: url(:/images/close_first.png);
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{

	
	background-image: url(:/images/close_after.png);
background-repeat:no-repeat;
background-position:center center;


}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>641</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton_newReport">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>15</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_newReport
{
    color:black;
    background-color:rgb(238 , 238 , 238);
    border: 1px solid rgb(183 , 183 , 183);
}

QPushButton#pushButton_newReport:hover
{
    color:black;
    background-color:rgb(228 , 240 , 250);
    border: 1px solid rgb(15 , 150 , 255);
}

QPushButton#pushButton_newReport:pressed
{
    color:black;
    background-color:rgb(204 , 228 , 247);
    border: 1px solid rgb(1 , 84 , 153);
    padding-left:3px;
    padding-top:3px;
}</string>
    </property>
    <property name="text">
     <string>新增周报</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>15</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>年份：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_year">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>15</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>15</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>时间段：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_period">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>15</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_name">
    <property name="geometry">
     <rect>
      <x>178</x>
      <y>16</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>16</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>姓名：</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>个人周报</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>641</width>
     <height>611</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(221, 245, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_dontSubmit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>710</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>显示未交人员</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sendTipMsg">
   <property name="geometry">
    <rect>
     <x>404</x>
     <y>710</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>一键发送提醒</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
